common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -8,7 +8,7 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
       % if response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${HTML(response_data['label'])}</legend>
```

Comment:
@robrap `response_data['label']` can be HTML and Text. Is this OK? What is the best approach for this scenario?